migration/migration.go

```diff
@@ -29,6 +29,7 @@ var logger = loggo.GetLogger("juju.migration")
 type StateExporter interface {
 	// Export generates an abstract representation of a model.
 	Export() (description.Model, error)
+	ExportPartial(cfg state.ExportConfig) (description.Model, error)
```

Comment:
This does not sound valid as we do not want to affect anything to do with migrations :D

Reply:
We don't need two interfaces. The `Backend` interface should just have the `ExportPartial` function.